scene/gui/view_panner.h

```diff
@@ -55,22 +65,30 @@ class ViewPanner : public RefCounted {
 
 	Ref<Shortcut> pan_view_shortcut;
 
-	Callable scroll_callback;
 	Callable pan_callback;
 	Callable zoom_callback;
 
 	void callback_helper(Callable p_callback, Vector<Variant> p_args);
 	ControlScheme control_scheme = SCROLL_ZOOMS;
 
 public:
-	void set_callbacks(Callable p_scroll_callback, Callable p_pan_callback, Callable p_zoom_callback);
+	void set_callbacks(Callable p_pan_callback, Callable p_zoom_callback);
 	void set_control_scheme(ControlScheme p_scheme);
 	void set_enable_rmb(bool p_enable);
 	void set_pan_shortcut(Ref<Shortcut> p_shortcut);
 	void set_simple_panning_enabled(bool p_enabled);
 
 	void setup(ControlScheme p_scheme, Ref<Shortcut> p_shortcut, bool p_simple_panning);
 
+	int get_scroll_speed() const;
+	void set_scroll_speed(int p_scroll_speed);
+
+	float get_scroll_zoom_factor() const;
+	void set_scroll_zoom_factor(float p_scroll_zoom_factor);
+
+	PanAxis get_pan_axis() const;
```

Comment:
Getter is not necessary if there is no property bound (and you don't use it anywhere). I'd remove it.

Reply:
Right, I don't mind removing it.